app/subscriber/src/features/my-reports/utils/reportInstanceContentInlineDisplay.ts

```diff
@@ -0,0 +1,26 @@
+import moment from 'moment';
+import { ContentTypeName } from 'tno-core';
+
+import { IReportInstanceContentForm } from '../interfaces';
+
+export const reportInstanceContentInlineDisplay = (row: IReportInstanceContentForm) => {
+  if (!row.content) return '';
+  if (row.content.contentType === ContentTypeName.AudioVideo) {
+    // AV Content format
+    console.log('xxx1', row.content.series);
```

Comment:
Remove

Reply:
I remember our lint flagged the log, but anyway, I will ensure next time it won't be in the code.